src/hooks/useHakukohdeSearch.ts

```diff
@@ -71,18 +80,30 @@ export const useHakukohdeSearchResults = (hakuOid: string) => {
     ],
   });
 
-  const { searchPhrase, withValintakoe } = useHakukohdeSearchParamsState();
+  const { searchPhrase, withValintakoe, withoutLaskenta } =
+    useHakukohdeSearchParamsState();
 
   const sortedHakukohteet = useMemo(() => {
-    const filteredHakukohteet = withValintakoe
+    let filteredHakukohteet = withValintakoe
```

Comment:
ei ole pakko ottaa tuota, mutta ajattelin kuitenkin ehdota alla olevaa
```suggestion
     const filteredHakukohteet = hakukohteet
    .filter((hakukohde) =>
      !withValintakoe || suodatustiedot?.[hakukohde.oid]?.hasValintakoe
    )
    .filter((hakukohde) =>
      !withoutLaskenta || !suodatustiedot?.[hakukohde.oid]?.laskettu
    );
```